src/controllers/itemDetails.js

```diff
@@ -415,30 +416,29 @@ define(['loading', 'appRouter', 'layoutManager', 'connectionManager', 'userSetti
                 }
             } else {
                 if (layoutManager.mobile) {
-                    html = '<h1 class="parentName" style="margin: .1em 0 .25em;">' + parentNameHtml.join('</br>') + '</h1>';
+                    html = '<h1 class="parentName" style="margin: 0.2em 0 0">' + parentNameHtml.join('</br>') + '</h1>';
```

Comment:
Maybe I'm missing something, but I don't understand why all the changes in this file are relevant for our aims in this PR.

Reply:
The CSS changes were regarding the title issues on the detail sticky, which has terrible margins at the moment. It's indeed completely unrelated to this pull request, but I combined them because a full pull request for five lines of changes seems excessive.

Reply:
Alright